tui/query_text_area.go

```
@@ -1,15 +1,20 @@
 	"github.com/gdamore/tcell/v2"
+	"github.com/nao1215/sqluv/domain/model"
 	"github.com/rivo/tview"
 )
 
 // queryTextArea represents a query input field.
 type queryTextArea struct {
 	*tview.TextArea
-	theme *Theme
+	theme          *Theme
+	candidates     []string // SQL reserved words, table names, and column names.
+	completionList *tview.List
```

golangci-lint (1.64.8)

field `completionList` is unused

(unused)